### PR TITLE
Test: Rename: ChargebackRateDetailCurrency -> Currency

### PR DESCRIPTION
**Pull request by lpichler**
https://github.com/ManageIQ/manageiq/pull/19538

### CODE REVIEW

**Comment by lpichler** [comment]
https://github.com/ManageIQ/manageiq/pull/19542

File: .travis.yml [hunk 1]
```diff
@@ -15,5 +15,31 @@ matrix:
 env:
   global:
   - REPOS=
+  - CORE_REPO=manageiq#19538
```
Comment (himdel) [inline .travis.yml]: should be `manageiq@19538` right?

Comment (agrare) [reply .travis.yml]: A PR should be # a commit/branch/tag should be @
Looks like the readme is wrong, we follow the standard GitHub short format.

Comment (agrare) [reply .travis.yml]: Also CORE_REPO vs GEM_REPOS are no longer necessary, they're combined into just REPOS

**Pull request closed without merging**